app/assets/stylesheets/modules/playdates.scss

```diff
@@ -0,0 +1,42 @@
+.create-playdates {
+  background: #788a6f;
+}
+
+.playdates {
+  background: url("abel.jpg") no-repeat center center fixed;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+}
+
+.create-bg {
+  background: url("poof.jpg") no-repeat center center fixed;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+}
+
+#window-styling {
+  padding-top: 2px;
+}
+
+.preferences {
+  background-color: rgba(255, 255, 255, 0.5);
+  margin-top: 2px;
+  padding-bottom: 10px;
+}
+
+.quote-box {
+  margin-left: 10px;
+  margin-right: 10px;
+  padding: 10px;
+}
+
+.new-shelter-image {
+  margin-left: 10px;
```

Comment:
Properties should be ordered margin-bottom, margin-left, margin-right, margin-top